bcmc/src/main/java/com/adyen/checkout/bcmc/internal/ui/model/BcmcComponentParamsMapper.kt

```diff
@@ -40,12 +48,23 @@ internal class BcmcComponentParamsMapper(
             isHolderNameRequired = isHolderNameRequired ?: false,
             shopperReference = shopperReference,
             isStorePaymentFieldVisible = isStorePaymentFieldVisible ?: false,
+            addressParams = AddressParams.None,
+            installmentParams = null,
+            kcpAuthVisibility = KCPAuthVisibility.HIDE,
+            socialSecurityNumberVisibility = SocialSecurityNumberVisibility.HIDE,
+            cvcVisibility = cvcVisibility ?: CVCVisibility.ALWAYS_HIDE,
+            storedCVCVisibility = StoredCVCVisibility.HIDE,
```

Comment:
Shouldn't this be configurable for BCMC as well?

Reply:
Added